src/converters/xml.cr

```diff
@@ -0,0 +1,88 @@
+module OQ::Converters::Xml
+  @@at_root : Bool = true
```

Comment:
This is a bit of a smell, it means you can't call methods of this module concurrently. Is there a way to pass this `at_root` value to all methods that need it so it's not shared at all?

Reply:
Yea I could probably just pass it between the methods versus making it a class var.

Reply:
Oh, I edited my comment and some part was gone. I forgot to say that this is probably a minor issue because this is a one-shot app meaning that you won't be running this concurrently, but it's still a good design pattern to avoid sharing stuff.

I thought about making it a class/struct but you'd still have the same problem that if someone uses that instance concurrently then the `@at_root` instance var would cause conflicts. I think there's no other way around passing it between methods. And instead of passing a lot of args you can consider creating a class holding these shared things and passing that instead.

But again, this is probably not needed for this kind of app 😊